conanfile.py

```diff
@@ -320,6 +324,32 @@ def build_configure(self):
             if self.settings.os == "Windows":
                 args.append('--enable-libmfx' if self.options.qsv else '--disable-libmfx')
 
+            if tools.cross_building(self.settings):
+                args.append('--enable-cross-compile')
+                if self.settings.os == "Android":
+                    args.append('--target-os=android')
```

Comment:
consider adding function matching `self.settings.os` into `--target-os`. this is needed not just for Android, iOS and others need this as well